index.html

```diff
@@ -897,12 +897,22 @@ <h5 class="text-center">Environmental Map (DEQ)</h5>
       <p class="muted">Search for site-specific information relating to DEQ programs including regulated environmental
         sites, facilities, and designations.</p>
     </div>
+    <div class="grid__col grid__col--1-of-2 text-center">
```

Comment:
since there are 3 columns now, this needs to match the upper stuff and be 1-of-3 in all the spots.